### DISCUSSION TITLE
How to enable HTTPS support for the HTTP request block in Dify.

### DISCUSSION THREAD

**Discussion by ismasher2000** [langgenius/dify#9473] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

My API backend must be based on HTTPS for security compliance purposes, but I cannot find a way to enable HTTPS support for the HTTP Request Block. After testing, it seems that the HTTP Request Block does not support HTTPS at all, which poses certain security risks.

**Comment by dingjiahui00** [comment]
I also want to know how to send https requests in http nodes
